### PR TITLE
Data gen and database

### PR DESCRIPTION
**Pull request by jgenso1690**
mongoose schema
generating fake data up to 100
getting get requests from different endpoints id# from 1- 100

### CODE REVIEW

File: public/bundle.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
+/******/ (function(modules) { // webpackBootstrap
```
Comment (hieuhho) [inline public/bundle.js:1]: add this to .gitignore

File: server/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+   var id = req.params;
+    
+    db.findOne({ 'features.id' : `${id.id}` }, function (err, feature) {
+    if(err) console.log('errordatA');
+    console.log('%s is the id', feature)
+    res.send(feature)
+   })
+})
+
+accordion.listen(port,() =>console.log(`Listening on port ${port}`));
```
Comment (hieuhho) [inline server/index.js:21]: add extra empty line to follow style guide

File: package.json [hunk 3]
```diff
@@ -22,12 +25,18 @@
     "babel-loader": "^8.1.0",
     "babel-watch": "^7.0.0",
     "webpack": "^4.43.0",
-    "webpack-cli": "^3.3.11"
+    "webpack-cli": "^3.3.11",
+    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor"
```
Comment (hieuhho) [inline package.json:29]: when you have time, change to airbnb style guide. Installation and settings in the dependencies doc

**Review by hieuhho — APPROVED** [review]
Left some notes, no big changes needed so approved for now.
Install the airbnb linter when you have the time.

File: server/index.js (hunk 2 above)
Comment (jgenso1690) [reply server/index.js:21]: I do have a extra line in my vs, I don't know why is not showing here. Thank you for noticing

**Pull request merged**